macroid-core/src/main/scala/macroid/Contexts.scala

```diff
@@ -51,7 +52,6 @@ object ContextWrapper {
 If you are inside Activity or Fragment, extend Contexts[Activity] or Contexts[Fragment],
 otherwise pass an instance of `FragmentManagerContext` from outside.
 Please note that for support fragments you need to extends Contexts[FragmentActivity]""")
-/** FragmentManager context */

```

Comment:
Now that you mentioned it, it’s indeed weird. Could you just add a few words to it, like “FragmentManager context, used to manipulate fragments within an Activity or another Fragment”?